app/views/content/a-day-in-the-life-of-a-teacher.md

```diff
@@ -17,79 +18,145 @@
+
+$q-phil-secondary$
+
+
+#### 11:15am morning break
+Teachers are sometimes responsible for break-time duties. This involves supervising pupils for their safety but it is an opportunity to interact with pupils outside of the classroom and build relationships. 
+
+#### 11:35am lessons 
+
+After break, pupils return to lessons with renewed energy, which can be harnessed and channeled into their work. During this time, teachers usually teach two more lessons before lunch.
+$q-jessica-science$
+
+
+### Lunch
+
+Lunch is an opportunity for teachers to have a well-earned break and go to the staff room to talk about how the morning has been. Many teachers also use this as an opportunity to get ahead on their marking. By tackling some books during this time, it can reduce the amount of marking needed to be done in the afternoon.
+
+$q-mike-secondary$
+
+
+### Afternoon
+
+#### 2:15pm afternoon lessons 
+
+After lunch, there is often just one more lesson to teach. Schools sometimes work the timetable this way because after lunch pupils are generally more excitable making it harder to teach.  
 
-A day in the life of a teacher is really varied, and no two days will be the same. Your day as a teacher will look different depending on whether you teach primary, secondary, or what subject you teach. Each school will have its own schedule and different responsibilities outside of teaching hours.
 
-Aside from classroom time, teaching is quite flexible. You can choose when you do your marking, planning, and admin. Teachers also get more holidays than other professions. Full-time teachers work for 195 days per year in school.
+#### 3:15pm pupils leave school
+When pupils leave school the work continues. Teachers generally spend this time marking work and planning upcoming lessons.
 
-----
+Some teachers will also run an afterschool club such as a football team, drama sessions or computer club. 
 
-#### 7:45 - Arrive at school
+$q-gavin-ict$
 
-Many teachers like to get to school around this time or before. They prepare for the day, get any equipment they need, and make sure everything’s ready for classes to start.
 
-$q-sarah-primary$
+#### 5:00pm leave school
 
-#### 8:20 - Staff meeting
+Depending on how the teacher likes to split their work, they will leave school between 4:30pm to 6:00pm. Some teachers will take home books to mark or carry on planning once they are at home.
 
-There may be staff meetings in the morning to catch up with all staff, year group, or faculty members.
+### Planning preparation and assessment time (PPA)
+Teachers are entitled 10% of their time off time table to plan work. Depending on the school, this could mean that you have a couple of lessons free next to each other or they could be spread across the week.
```

Comment:
```suggestion
Teachers are entitled to 10% of their time without lessons, so they can plan work. Depending on the school, this could mean that you have a couple of free periods next to each other or they could be spread across the week.
```
I found this a bit difficult to read/understand. Have tried to tweak.